elasticsearch/install.sh

```diff
@@ -23,7 +23,12 @@ mkdir /elasticsearch
 mkdir -p $ES_CONF
 chmod -R og+w $ES_CONF ${ES_HOME} ${HOME} /elasticsearch
 chmod -R o+rx /etc/elasticsearch
-chmod +x ${ES_HOME}/plugins/openshift-elasticsearch/sgadmin.sh
+chmod +x ${ES_HOME}/plugins/search-guard-6/tools/sgadmin.sh
+chmod o+r /etc/sysconfig/elasticsearch
```

Comment:
we can merge this for now, but can you please figure out what's up with all these permission changes and what's needed to avoid them?

Reply:
It seems like when we install Elasticsearch, it will create a user called `elasticsearch` that owns those directories (i.e. `/etc/sysconfig/elasticsearch`, `/var/log/elasticsearch`). In `run.sh`, we do `exec /bin/elasticsearch` as USER 1000 which causes `permission denied`. I guess if we can run as `elasticsearch` we won't have to change those permissions.

Reply:
@longnguyennn can you please create a ticket for that?